client/src/components/profile/components/Certifications.js

```diff
@@ -38,7 +38,7 @@ const propTypes = {
 
 function renderCertShow(username, cert) {
   return cert.show ? (
-    <Fragment key={cert.title}>
+    <>
```

Comment:
The shorthand cannot replace the Fragment with the `key` property.

Reply:
Fixed